lib/proto/proto.go

```diff
@@ -1232,78 +1230,3 @@ func (x EnumValueDescriptor) CompareSameType(op syntax.Token, y_ starlark.Value,
 		return false, fmt.Errorf("%s %s %s not implemented", x.Type(), op, y_.Type())
 	}
 }
-
-// A Bytes is an immutable sequence of bytes.
-// It is comparable, iterable, indexable, and sliceable.
-//
-// (In go.starlark.net, text Strings are also byte strings,
-// but we shouldn't rely on that.
-// See https://github.com/bazelbuild/starlark/issues/112.)
-type Bytes string
```

Comment:
Could this be a type alias to the new type? Not sure if this is used yet.

Reply:
> Could this be a type alias to the new type?

No. syntax.Bytes is a part of the syntax package, whereas starlark.Bytes (a value) is part of the interpreter. Even if we wanted them to be the same, the dependency is in the wrong direction.

> Not sure if this is used yet.

It's the type of the "payload" of syntax.Literal when TokenKind==BYTES. Having just said that, I wonder if we can use string since the TokenKind should give us the information need to discriminate. [checks...] Ah, the reason is that compile.Program.Constants[i] needs to discriminate based on type alone. I suppose it could define a private Bytes type. Done.

Reply:
Ah, I meant to ask if this could be a type alias for `starlark.Bytes`, just to avoid a breaking change to users of `lib/proto`. Not sure if anyone is using this type though, since `lib/proto` landed recently.